packages/core/lib/modules/wallet.test.js

```diff
@@ -0,0 +1,45 @@
+import {Wallet} from './wallet';
+
+// TODO: Test events
+
+describe('ApiModule', () => {
```

Comment:
more unhappy test cases here too

Reply:
should tests and app code be in separate folders? I've seen a pattern with `__tests__` as a sub-folder pretty commonly in React projects before

Reply:
The sdk can be used in nodejs as well, and as of now it's not tight to react.

Reply:
we can provide additional packages with some react hooks, which will be helpful in react projects

Reply:
even for node modules, would it make sense to keep test code separate?